tests/virt/vmmigration_test.py

```diff
@@ -268,6 +269,42 @@ def test_post_copy_status(self):
         assert stats['status'] == vmstatus.PAUSED
 
 
+class TestMigrationTimeout(TestCaseBase):
+
+    def test_disk_timeout_needed(self):
+        with fake.VM(xmldevices=read_data('disk_devices.xml'),
+                     metadata=read_data('disk_devices_meta.xml')) as testvm:
+            timeout = testvm._disk_preparation_timeout()
+            assert timeout == 4.0
```

Comment:
I'm not sure that this assert will be the nice pytest assert, because you use a legacy
unittest based class. Then when the test fail it may give you unhelpful output.